yarn.lock

```diff
@@ -8767,7 +8767,7 @@ strip-indent@^2.0.0:
 
 strip-indent@^3.0.0:
   version "3.0.0"
-  resolved "https://registry.yarnpkg.com/strip-indent/-/strip-indent-3.0.0.tgz#c32e1cee940b6b3432c771bc2c54bcce73cd3001"
+  resolved "https://registry.yarnpkg.com/strip-indent/-/strip-indent-3.0.0.tgz#c32e1cee940b6b3432c771bc2c54bcce73cd5000"
```

Comment:
This change was giving me a `Integrity check failed for "strip-indent"` when installing locally

Reply:
ah, a mistake from changing the port locally for myself!